app/controllers/healthcare_controller.rb

```diff
@@ -22,6 +23,12 @@ def update
 
   private
 
+  def update_document_workflow
```

Comment:
We're going to add a few more of these calls to our controllers soon. Are you happy to wait and see how that pans out before refactoring?